common/app/views/support/AmpAd.scala

```diff
@@ -0,0 +1,43 @@
+package views.support
+
+import model.Tag
+import model.Article
+import play.api.libs.json.Json
+
+object AmpAd {
```

Comment:
Sounds good. Would it also be appropriate to write tests for these classes, do you think?